Contentful.Core/ContentfulClient.cs

```diff
@@ -643,7 +643,7 @@ private string BuildSyncQuery(SyncType syncType = SyncType.All, string contentTy
 
         private async Task<HttpResponseMessage> Get(string url, CancellationToken cancellationToken)
         {
-            return await SendHttpRequest(url, HttpMethod.Get, _options.DeliveryApiKey, cancellationToken).ConfigureAwait(false);
+            return await SendHttpRequest(url, HttpMethod.Get, _options.UsePreviewApi ? _options.PreviewApiKey : _options.DeliveryApiKey, cancellationToken).ConfigureAwait(false);
```

Comment:
So, if preview API key is `nil` what will happen? just a response saying unauthorized? any issues with serializing the request to be worried about?